pkg/pool-stable/contracts/ComposableStablePool.sol

```diff
@@ -903,6 +946,31 @@ contract ComposableStablePool is
         return (bptAmountIn, amountsOut);
     }
 
+    function _computeProportionalAmountsOut(
```

Comment:
This should probably _not_ be here.

When merging master back to the branch to solve the conflicts, the build breaks without this.
This function was implemented in `RecoveryMode`, and is similar to `_doRecoveryModeExit` (except that the latter expects balances including BPT, and the API is different).

I just added it so that we can start the review, but this should be addressed before merging.